src/content-feature.js

```diff
@@ -98,16 +100,7 @@ export default class ContentFeature {
      */
     get messagingContext () {
```

Comment:
I've left this in as DuckPlayer uses it, however I think it should all be hidden within `get messaging`